avaframe/runStandardTests.py

```diff
@@ -50,8 +50,8 @@
 for avaDir in standardNames:
 
     # get path to executable
-    cfgSamosAT = cfgUtils.getModuleConfig(com1DFA)
-    samosAT = cfgSamosAT['GENERAL']['samosAT']
+    cfgCom1DFA = cfgUtils.getModuleConfig(com1DFA)
+    com1Exe = cfgCom1DFA['GENERAL']['com1Exe']
 
```

Comment:
Question: should the avalancheDir be cleaned with cleanSingleAvaDir?

Reply:
yes. cleanSingleAva needs directory in which Ouputs , Work etc (potentially) live...

Reply:
Would also be useful to add this here in the runStandardTests.py

Reply:
yes, definitely